utils/buildJsonResults.js

```diff
@@ -28,14 +31,31 @@ module.exports = function (report, appDirectory, options) {
       return;
     }
 
+    // If the usePathForSuiteName option is true and the
+    // suiteNameTemplate value is set to the default, overrides
+    // the suiteNameTemplate.
+    if(options.usePathForSuiteName === "true"
+      && options.suiteNameTemplate === constants.TITLE_VAR) {
+
+      options.suiteNameTemplate = constants.FILEPATH_VAR;
+    }
+
+    // Build variables for suite name
+    const filepath = suite.testFilePath.replace(appDirectory, '');
```

Comment:
Replace with `const filepath = path.dirname(appDirectory);`

Reply:
Actually I think I misread this. You are trying to show the relative file path from the app root? If so then do ```const filepath = path.relative(appDirectory, suite.testFilePath);```

Reply:
I was trying to make sure that setting `suiteNameTemplate="{filepath}"` would have the same result as `usePathForSuiteName="true"`. Changing to use `path.relative(...)` isn't having the same behavior when the `appDirectory` is set as before. 

The example from the tests is 

```shell
buildJsonResults › should return the proper name from testFilePath when usePathForSuiteName is "true"; with appDirectory set

    expect(received).toBe(expected)

    Expected value to be (using ===):
      "/__tests__/foo.test.js"
    Received:
      "/path/to/test/__tests__/foo.test.js"
```

Reply:
Ah ok fair point. Since the path is not relative then let's switch it to path.dirname(appDirectory). That should provide the result needed?

Reply:
I tried playing around with this a little bit, but couldn't get any of the previous tests to pass and I don't want to change the existing behavior for having the path be the name of the rendered testsuite. 

I'm happy to change it, but it feels like that would technically be a breaking change (although its probably not a big deal in reality), and therefore doesn't belong in this PR (?)

Let me know your thoughts on that. Thanks for being patient with me on this :)

Reply:
Ok cool - I'm ok merging it as-is and we can always revisit later on to tweak some things if needed. Nice work!